docs/content/ingestion/batch-ingestion.md

```diff
@@ -161,10 +161,19 @@ The tuningConfig is optional and default parameters will be used if no tuningCon
 |ignoreInvalidRows|Boolean|Ignore rows found to have problems.|no (default == false)|
 |combineText|Boolean|Use CombineTextInputFormat to combine multiple files into a file split. This can speed up Hadoop jobs when processing a large number of small files.|no (default == false)|
 |useCombiner|Boolean|Use Hadoop combiner to merge rows at mapper if possible.|no (default == false)|
-|jobProperties|Object|A map of properties to add to the Hadoop job configuration.|no (default == null)|
+|jobProperties|Object|A map of properties to add to the Hadoop job configuration, see below for popular options.|no (default == null)|
 |buildV9Directly|Boolean|Build v9 index directly instead of building v8 index and converting it to v9 format.|no (default = false)|
 |numBackgroundPersistThreads|Integer|The number of new background threads to use for incremental persists. Using this feature causes a notable increase in memory pressure and cpu usage but will make the job finish more quickly. If changing from the default of 0 (use current thread for persists), we recommend setting it to 1.|no (default == 0)|
 
+#### Job properties
+
+The following properties can be used to tune how the MapReduce job is configured by overriding default Hadoop/YARN/Mapreduce configurations:
+
+|Property name|Type|Description|
+|-------------|----|-----------|
+|mapreduce.job.user.classpath.first|String|Use Druid classpath instead of Hadoop classpath for common libraries like [Jackson](https://github.com/FasterXML/jackson) (required with [Cloudera Hadoop distribution](../operations/other-hadoop.html)) when set to `"true"`.|
```

Comment:
In my case, it helped me for batch ingestion without touching the code but I guess mileage can vary a lot with various Hadoop distributions :disappointed:.

What about describing the job properties configuration as an expert feature like it is often found in Lucene. That is something like "Please note that this is an expert feature and should not be used without a deep understanding of Hadoop and Java class loading management" and "see below for details" instead of "see below for popular options".